FlowCrypt/src/test/java/com/flowcrypt/email/security/pgp/PgpKeyTest.kt

```diff
@@ -6,11 +6,14 @@
 package com.flowcrypt.email.security.pgp
 
 import com.flowcrypt.email.BuildConfig
+import com.flowcrypt.email.extensions.kotlin.toHexString
 import com.flowcrypt.email.security.model.Algo
 import com.flowcrypt.email.security.model.KeyId
 import com.flowcrypt.email.security.model.PgpKeyDetails
 import org.junit.Assert.assertEquals
+import org.junit.Ignore
```

Comment:
Unused import

Reply:
Would it be easier to integrate a Kotlin linter into the Semaphore tests, that way we don't need to depend on manual review to catch little things like this?

Reply:
Agree - there is #584 #1122 #545 . We should use software for this, not humans.

Reply:
I remember we have a task for this, even one assigned to me #1122. So I will be working on it when more tasks are finished.